src/main/java/it/gov/pagopa/payhub/activities/dto/CartDTO.java

```diff
@@ -0,0 +1,115 @@
+package it.gov.pagopa.payhub.activities.dto;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import java.io.Serializable;
+import java.math.BigDecimal;
+import java.util.Date;
+@Data
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor
+public class CartDTO implements Serializable {
```

Comment:
do we really need this entity on the new model?
if it's not required, don't create it